src/App.tsx

```diff
@@ -1,11 +1,15 @@
 import { AboutMeComponent } from '@/components/about/AboutMe.component.tsx';
+import { FooterComponent } from '@/components/footer/Footer.component.tsx';
+import { HeaderComponent } from '@/components/header/Header.component.tsx';
 
 import './App.css';
 
 function App() {
     return (
         <>
+            <HeaderComponent />
             <AboutMeComponent />
```

Comment:
For wide screens (2000px+), center the content of the "about" section on the page.

Reply:
@serhii-tsivinskyi-moc done